app/controllers/builds_controller.rb

```diff
@@ -92,8 +92,18 @@ def edit_build_params
     params.require(:build).permit(:label, :description)
   end
 
+  def extra_docker_build_params
+    hash = {}
+    Samson::Hooks.fire(:extra_build_params).each do |p|
+      hash[p] = params[p].present?
```

Comment:
this sets true to some keys ... looks strange ... could merge extra_build_params ?